projects/igniteui-angular/src/lib/grids/grid/grid-summary.spec.ts

```diff
@@ -27,6 +27,8 @@ import { GridSummaryCalculationMode } from '../common/enums';
 import { IgxNumberFilteringOperand, IgxStringFilteringOperand } from '../../data-operations/filtering-condition';
 import { SortingDirection } from '../../data-operations/sorting-expression.interface';
 import { DropPosition } from '../moving/moving.service';
+import { DatePipe, registerLocaleData } from '@angular/common';
+import localeFR from '@angular/common/locales/fr';
```

Comment:
In this PR: #9182 the `registerLocaleData` call is being moved to the `test.ts` file, instead of having to call it in each file individually. I would wait for that PR to get merged and then remove the specific locale registration from this test.

Reply:
@mtsvyatkova I'm doing the merge now and I will resolve my own comment :)